security.md

```diff
@@ -5,77 +5,143 @@ css_id: security
+* **Slow retrieval attacks**. An attacker responds to clients with a very
+slow stream of data that essentially results in the client never continuing
+the update process.
 
-* **Mix-and-match attacks**. An attacker presents clients with a view of a repository that includes files that did not exist together on the repository at the same time. This can result in, for example, outdated versions of dependencies being installed.
+* **Extraneous dependencies attacks**. An attacker indicates to clients that
+ in order to install the software they wanted, they also need to install
+ unrelated software. This unrelated software can be from a trusted source
+ but may have known vulnerabilities that are exploitable by the attacker.
 
-* **Wrong software installation**. An attacker provides a client with a trusted file that is not the one the client wanted.
+* **Mix-and-match attacks**. An attacker presents clients with a view of a
+repository that includes files that did not exist together on the repository
+at the same time. This can result in, for example, outdated versions of
+dependencies being installed.
 
-* **Malicious mirrors preventing updates**. An attacker in control of one repository mirror is able to prevent users from obtaining updates from other, good mirrors.
+* **Wrong software installation**. An attacker provides a client with a
+ trusted file that is not the one the client wanted.
 
-* **Vulnerability to key compromises**. An attacker who is able to compromise a single key or less than a given threshold of keys can compromise clients. This includes relying on a single online key (such as only being protected by SSL) or a single offline key (such as most software update systems use to sign files).
+* **Malicious mirrors preventing updates**. An attacker in control of one
+repository mirror is able to prevent users from obtaining updates from other,
+good mirrors.
 
-## Design Concepts
+* **Vulnerability to key compromises**. An attacker who can compromises the one key
+in a single key system, or less than a given threshold of keys, can compromise
+clients. These attacks can occur whether the client relies on a single online
+key (if only being protected by SSL) or a single offline key (if protected by
+most software update systems that use keysigning).
 
-The design and implementation of TUF aims to be secure against all of the above attacks. A few general ideas drive much of the security of TUF.
+## Security Design Principles
 
-For the details of how TUF conveys the information discussed below, see the
+To ensure systems are secure against all of the above attacks, the design and
+implementation of TUF relies on a few basic concepts.
+For details of how TUF conveys the information discussed below, see the
 [Metadata documentation](/metadata.html).
```

Comment:
suggestion: "... see the Metadata page" or "see metadata."

Reply:
fixed